hawkrest/__init__.py

```diff
@@ -39,16 +71,7 @@ def authenticate(self, request):
             # This was added as a convenient way to run the apk-signer
             # test suite but it's probably the wrong way to do it.
             log.warn('Hawk authentication disabled via settings')
-            return on_success
-
-        lookup_function = lookup_credentials
-        settings_lookup_name = getattr(
-            settings,
-            'HAWK_CREDENTIALS_LOOKUP',
-            None
-        )
-        if settings_lookup_name:
-            lookup_function = import_string(settings_lookup_name)
+            return self.hawk_user_lookup(request, None)
```

Comment:
I think this is the wrong thing to do. It should just return `(HawkAuthenticatedUser(), None)` because otherwise the `None` may trigger a `KeyError` for custom lookup functions. Well, argh, this whole `SKIP_HAWK_AUTH` thing is stupid. I filed [this](https://github.com/kumar303/hawkrest/issues/35) to remove it (but don't feel like you need scope creep this patch for it).